x-pack/plugins/cloud/public/plugin.tsx

```diff
@@ -436,32 +313,6 @@ export class CloudPlugin implements Plugin<CloudSetup> {
   }
 }
 
-/** @internal exported for testing */
-export const loadUserId = async ({
-  getCurrentUser,
-}: {
-  getCurrentUser: () => Promise<AuthenticatedUser>;
-}) => {
-  try {
-    const currentUser = await getCurrentUser().catch(() => undefined);
-    if (!currentUser) {
-      return undefined;
-    }
-
-    // Log very defensively here so we can debug this easily if it breaks
-    if (!currentUser.username) {
-      // eslint-disable-next-line no-console
-      console.debug(
-        `[cloud.analytics] username not specified. User metadata: ${JSON.stringify(
-          currentUser.metadata
-        )}`
-      );
-    }
-
-    return currentUser.username;
-  } catch (e) {
-    // eslint-disable-next-line no-console
-    console.error(`[cloud.analytics] Error loading the current user: ${e.toString()}`, e);
-    return undefined;
-  }
-};
+function sha256(str: string) {
+  return new Sha256().update(str, 'utf8').digest('hex');
+}
```

Comment:
Existing tests validate that using this instead of the `js-sha256` library gets the same result.

Reply:
Resolves #130147